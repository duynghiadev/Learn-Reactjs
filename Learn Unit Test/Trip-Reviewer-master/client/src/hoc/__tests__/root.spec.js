import React from 'react';
import { create } from 'react-test-renderer';

import Root from '../root';

const mockComponent = props => <Root {...props} />;

describe('<Root />', () => {
  it('should render component', () => {
    const props = {
      children: <div>test</div>,
    };
    const tree = create(mockComponent(props)).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
