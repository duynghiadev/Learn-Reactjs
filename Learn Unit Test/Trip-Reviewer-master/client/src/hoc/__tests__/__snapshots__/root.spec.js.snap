// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Root /> should render component 1`] = `
<div>
  test
</div>
`;
